docs/tutorials/writing_a_series.rst

```diff
@@ -0,0 +1,112 @@
+
+Here's an example series file. Let's call it ``sanity_tests.yaml`` and include
+tests that are normally run just to make sure the machine works.
+
+.. code-block:: yaml
+
+  # This is the body of the series file. This is where we define test sets,
+  # which are groups of tests that we want to apply the same configurations to.
+  series:
+
+    # We'll call this test set 'front_end_tests' and run all the tests that
+    # don't run on compute nodes with this set.
+    front_end_tests:
+
+      # List names of tests
+      tests:
+        - check_mounts
+        - ping_compute_nodes
+        - check_commands.front_end
+
+      # List modes that need to be applied to this set of tests
+      modes:
+        - mode1
+        - mode2
+
+      # Define conditions. These will be addd to the only_if/not_if conditions
+      # that already exist in each test, if any.
+      only_if:
+        "{{sys.system_name}}": "machine"
+
+      # Don't run this set if the week day is Friday
+      not_if:
+        "{{pav.weekday}}": "Friday"
+
+    individual_node_tests:
+
+      # This group of tests depends on the previous set ('front_end_tests').
+      # Because 'depends_pass' is set to 'True', this set requires that all the
+      # tests in the set this set depends on must PASS in order for this set
+      # to be run. The default is that 'depends_pass' is 'False', which means
+      # the tests in the set this set depends on need only complete (regardless
+      # of whether or not they PASS) in order for this set to run.
+      depends_on: ['front_end_tests']
+      depends_pass: True
+
+      tests:
+        - stream.per_node
+
+    mp_tests:
+
+      modes: ['mode5']
+
+      depends_on: ['front_end_tests']
+      depends_pass: True
+
+      tests:
+        - stream.mpi
+        - supermagic.basic
+
+  # if 'ordered' is set to 'True', then Pavilion will automatically run this
+  # series such that each test set depends on the set before that.
+  ordered: False
+
+  # The total number of tests that will be run or scheduled at any given time.
+  # There is no default setting
+  simultaneous: 3
+
+  # series-level modes: These modes will be applied to all sets and therefore
+  # all tests
+  modes: ['mode3', 'mode4']
+
+  # We want this series to run continuously. The default is 'False', which means
+  # the series will run (or attempt to run) each set once only.
+  restart: True
+
+Running, Monitoring, and Cancelling the Series
+----------------------------------------------
+
+Run `pav show series` to show the list of possible series to run.
+
+To run a series file, the user must use the command
+``pav series <series_name>``. To run the series above, we would run
+``pav series sanity_tests``. This will output instructions on how to check the
+status of and kill the series.
+
+.. code-block:: text
+
+  $ pav series sanity_tests
+  Started series s19. Run `pav status s19` to view status. PGID is 9102.
+  To kill, use `kill -15 -9102` or `pav cancel s19`.
```

Comment:
I think just give the `pav cancel` instructions.  I don't want people getting in the habit of having to run kill manually (or remember what signal to send.

Reply:
The series PGID gets written to a file after the series subprocess is kicked off. The PGID needs to be written to a file in the series directory so that `pav cancel` knows what PGID to kill. 

In the event the PGID isn't able to be written to a file, I catch this error, and give the user only the `kill -15 -PGID` since `pav cancel s<ID>` will not work. 

Should I change is so that if it _does_ get written to a file, I only give the `pav cancel` instructions and give the instructions on how to manually kill the job _only_ when the PGID isn't written to a file? Or should I change it so that if it the PGID isn't written to a file I kill the subprocess entirely and not have any of the tests run?